zppy/templates/e3sm_diags.bash

```diff
@@ -101,19 +101,9 @@ create_links_ts()
       YYYY=`printf "%04d" ${year}`
       for file in ${ts_dir_source}/${v}_${YYYY}*.nc
       do
-        # Add this time series file to the list of files for cdscan to use
-        echo ${file} >> ${v}_files.txt
+        cp ${file} ${ts_dir_destination}/${v}_${YYYY}*.nc
```

Comment:
I suppose `${ts_dir_destination}/` isn't strictly necessary, since we have `cd ${ts_dir_destination}` above.